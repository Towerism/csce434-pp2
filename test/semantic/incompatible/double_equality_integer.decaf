int asdf() {
    int a;
    double b;
    a == b;
}