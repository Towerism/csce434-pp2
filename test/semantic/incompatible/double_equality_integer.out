
*** Error line 4.
    a == b;
      ^^
*** Incompatible operands: int == double

